lib/ClaimVerifier.ts

```diff
@@ -33,44 +30,46 @@ export class ClaimVerifier {
   }
 
   /**
-   * @description Checks that claim represents role and was
+   * @description Checks that credential represents role and was
    * issued by issuer required by role definition
    *
-   * @param claim: role claim
+   * @param credential: Role EIP191Jwt
    */
-  private async verifyRole(claim: Required<OffchainClaim>): Promise<{
-    name: IRoleDefinition['roleName'];
-    namespace: OffchainClaim['claimType'];
+  private async verifyRole(credential: Required<RoleEIP191JWT>): Promise<{
+    name: IRoleDefinitionV2['roleName'];
+    namespace: RoleEIP191JWT['payload']['claimData']['claimType'];
   } | null> {
     // if (!(await this.verifySignature(claim))) {
     //   return null;
     // }
 
-    const role = await this.getRoleDefinition(claim.claimType);
-    if (!role) {
-      return null;
-    }
-
-    if (role.version !== claim.claimTypeVersion) {
+    if (!credential?.payload) {
```

Comment:
`credential isn't optional, so we don't need to null-check it
```suggestion
    if (credential?.payload) {
```

Reply:
Aah, right!